client/src/components/DataFiles/DataFilesModals/DataFilesManageProjectModal.jsx

```diff
@@ -116,6 +116,7 @@ const DataFilesManageProjectModal = () => {
         </ModalHeader>
```

Comment:
Probably not a part of this PR:
Opening the "Add Project" modal does not reset modal state. e.g. if you add a project with a user, then open the "Add Project" modal again to add a new project, the previously added team member is present

Reply:
I don't think this is a new bug, we should open a new ticket for it if you think it's a priority.

Reply:
https://jira.tacc.utexas.edu/browse/FP-1636